dev/bitcoin/rtl.tf

```diff
@@ -6,6 +6,7 @@ resource "helm_release" "rtl" {
   dependency_update = true
 
   depends_on = [
-    helm_release.lnd
+    helm_release.lnd,
+    helm_release.loop
```

Comment:
Do we need this to solve a race condition? Are we seeing the same problem here as in deployments?

Reply:
Didn't have a race condition before in dev, but would be good to follow the better practice we learnt from production. Also good for testing as github actions are more similar this way to production.